plugin/minisnip.vim

```diff
@@ -10,6 +10,7 @@ let g:minisnip_trigger = get(g:, 'minisnip_trigger', '<Tab>')
 let g:minisnip_startdelim = get(g:, 'minisnip_startdelim', '{{+')
 let g:minisnip_enddelim = get(g:, 'minisnip_enddelim', '+}}')
 let g:minisnip_evalmarker = get(g:, 'minisnip_evalmarker', '~')
+let g:minisnip_skipmarker = get(g:, 'minisnip_skipmarker', '`')
```

Comment:
How about we just use the evalmarker as the skipmarker?
Chances are you never want to purposely <tab> into an evalmarker's contents so we should skip it anyway.
Thoughts?

Reply:
I sometimes use the backrefmarker to set a default value, which I might want to change. For example I use in latex:
```
\begin{figure}
  \centering
      \includegraphics[{{+width=\textwidth+}}]{{{+figure+}}}
  \caption{{{+Caption+}}}
  \label{fig:{{+~\~2+}
\end{figure}
```
Therefore I would prefer not to use the evalmarker as the skipmarker. I think having the possibility to explicitly decide whether it should be skipped or not is a good feature.

Perhaps we could do it the other way round. If there is the `evalmarker` the placeholder is skiped, unless a `donotskipmarker` is present?

Reply:
I like the idea of a donotskip as I think by default the majority of eval snippets will be things you just want evaluating.

Thanks for the PR, looks good so far :)